install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -13,6 +13,13 @@ security:
   enabled: true
   omitPostInstall: true
 
+#
+# Gateways Configuration, refer to the charts/gateways/values.yaml
+# for detailed configuration
+#
+gateways:
+  enabled: false
+
```

Comment:
Please add the ISTIO_META_NETWORK thing here.

Reply:
the constructive suggestion is ^^^ .. See how its done for the other multicluster thing

Reply:
I don't agree with the approach of defining the value in two places but pushed your requested changes.